api/Window.json

```diff
@@ -1321,41 +1321,54 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/convertPointFromNodeToPage",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": true,
+              "version_removed": true,
```

Comment:
I can confirm that these were once there and then removed, by me :)

https://chromium.googlesource.com/chromium/src/+/ed6edbcb38b468c0168c13ebe13fa2910fe32a41

I think that was Chrome 38, but OK to fill out later as suggested.